src/test/java/com/plaid/client/integration/RecipientCreateTest.java

```diff
@@ -15,10 +16,10 @@
 public class RecipientCreateTest extends AbstractIntegrationTest {
 
   /**
-   * Utility method that creates a payment recipient.
+   * Utility method that creates a payment recipient with an iban.
    * Used by other integration tests to set up.
    */
-  public static Response<RecipientCreateResponse> createRecipient(PlaidClient client) throws Exception {
+  public static Response<RecipientCreateResponse> createRecipientWithIban(PlaidClient client) throws Exception {
```

Comment:
Curious why we want the IBAN implementation detail in the name if we no longer have the `createRecipient` method?

Reply:
We need to create a recipient and we can either create a recipient with an IBAN, BACS, or both